package.json

```diff
@@ -22,18 +22,21 @@
     "@nuxtjs/i18n": "8.0.0-rc.4",
     "@pinia/nuxt": "^0.4.11",
     "@playwright/test": "^1.37.1",
+    "@testing-library/vue": "^7.0.0",
```

Comment:
Have to install this one manually. For some reason it cannot be auto picked up by "nuxt-vitest" package